voila/handler.py

```diff
@@ -192,6 +192,30 @@ async def get(self, path=None):
             self.write(html)
             self.flush()
 
+    def render_template(self, name, **ns):
```

Comment:
Since `render_template` is not used directly in `get`, can you add some docstrings explaining why we need to add this method?

Reply:
Yes! Thanks for the review.